### DISCUSSION TITLE
能否添加  OpenWrt 路由器  mips_24kc 架构 的版本

### DISCUSSION THREAD

**Discussion by glenlau228** [messense/aliyundrive-webdav#208] [Q&A] [answered]
路由器的CPU 架构是 mips_24kc 10

**Comment by messense** [comment]
什么路由器？

**Comment by glenlau228** [comment]
网件 R6100

**Comment by messense** [comment, marked as answer]
[aliyundrive-webdav_1.0.0_mips_24kc.ipk.zip](https://github.com/messense/aliyundrive-webdav/files/7759637/aliyundrive-webdav_1.0.0_mips_24kc.ipk.zip)
